build_deploy.rhel.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Since the migration is on our plate, shall we start using the ubi as base image and start use of existing Makefile based build setup? I raised this concern with Moar when he was doing it for backbone, he told that we can do it according to our wish, but he couldn't do due to time constraints.

Reply:
I had thought of this but I think it will take a lot of time. There is definitely value in it. 
I am not convinced of the Makefile approach. If this is all we want to do in the build do we need a Makefile. 
I am assuming that the other stuff like pylint, pydoc and test scripts will not be needed going forward.

Reply:
Moved to UBI image and removed RHEL file.